assignment/assignment-bundles/resources/assignment.properties

```diff
@@ -903,8 +903,10 @@ content_review.accepted.types.delimiter=,
 content_review.accepted.types.lparen=(
 content_review.accepted.types.rparen=)
 
-noti.releaseresubmission.subject.content=Email notification for assignment with resubmissions allowed
-noti.releaseresubmission.text=Your submission to assignment "{0}" has been corrected and it allows resubmissions. Please go to {1} to view details.
+noti.releaseresubmission.subject.content=Email notification for assignment
```

Comment:
'Email notification' is redundant in a subject line too. Ideally I'd aim for something like:
"Your assignment has been graded", or such... 'Graded' implies there was a grade, 'corrected' implies there were corrections, 'released' is already defined for when the instructor posts the assignment. Marked perhaps?